content/documentation/staging/tutorial/05-connect/07-mirroring.adoc

```diff
@@ -0,0 +1,75 @@
+---
+title: Mirroring
+weight: 56
+---
+:icons: font
+:sectlinks:
+
+This tutorial has shown several scenarios where Istio can route traffic to different versions within the context to compare versions and evaluate which one is working better.
+
+The link:../tutorial/#03-traffic-shifting[Traffic Shifting] step was focused on *travels* service adding a new *travels-v2* and *travels-v3* workloads
+and the link:../tutorial/#04-tcp-traffic-shifting[TCP Traffic Shifting] showed how this scenario can be used on TCP services like *mysqldb* service.
+
+Mirroring (or shadowing) is a particular case of the Traffic Shifting scenario when the proxy sends a copy of live traffic to a mirrored service.
+
+The mirrored traffic happens out of band of the request path for a primary service allowing to test new services in production environments with minimal risk.
```

Comment:
The "out of band of the request path" expression is taken from [1].

It means that "mirrored" traffic is a copy, like the "tap" pattern in classic SOA.

Imagine 2 services: A -> B, and you have now a mirror called B-Mirror, so A request to B are copied to B-Mirror, but B-Mirror responses are not caught by A because the traffic is "out of band" where the "request path" is A -> B, but not A -> B-Mirror.

Basically a Mirror is a live load generator which allows you to get real requests.

[1] https://istio.io/latest/docs/tasks/traffic-management/mirroring/